app/proguard-rules.pro

```diff
@@ -17,56 +17,32 @@
 # keep setters in Views so that animations can still work.
 # see http://proguard.sourceforge.net/manual/examples.html#beans
 -keepclassmembers public class * extends android.view.View {
-   void set*(***);
-   *** get*();
+    void set*(***);
+    *** get*();
 }
 # We want to keep methods in Activity that could be used in the XML attribute onClick
 -keepclassmembers class * extends android.app.Activity {
-   public void *(android.view.View);
+    public void *(android.view.View);
 }
 # For enumeration classes, see http://proguard.sourceforge.net/manual/examples.html#enumerations
 -keepclassmembers enum * {
     public static **[] values();
     public static ** valueOf(java.lang.String);
 }
 -keepclassmembers class * implements android.os.Parcelable {
-  public static final android.os.Parcelable$Creator CREATOR;
-}
--keepclassmembers class **.R$* {
-    public static <fields>;
+    public static final ** CREATOR;
 }
 
 # TUSKY SPECIFIC OPTIONS
 
-## for okhttp
```

Comment:
All those library-specific rules can be deleted because they are now included directly in the libraries